onnxruntime/test/onnx/main.cc

```diff
@@ -403,8 +403,6 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
       {"unique_sorted_axis_3d", "Unique not implemented yet"},
       {"unique_sorted_axis", "Unique not implemented yet"},
       {"unique_sorted_with_negative_axis", "Unique not implemented yet"},
-      {"gather_elements_1", "not implemented yet"},
```

Comment:
Gather elements is different from Gather. I will remove it from exclusion in my PR #1735. Please keep it in the list for now.

Reply:
Add it back